src/components/clusterConfiguration/HostsTable.tsx

```diff
@@ -41,7 +41,7 @@ type OpenRows = {
 };
 
 const columns = [
-  { title: 'Serial Number', transforms: [sortable], cellFormatters: [expandable] },
+  { title: 'Host name', transforms: [sortable], cellFormatters: [expandable] },
```

Comment:
I think this should be called 'Hostname'

Reply:
done